### DISCUSSION TITLE
G14 2024 (GA403UI) fan control - strange behavior...

### DISCUSSION THREAD

**Discussion by Noisrevid** [seerge/g-helper#2169] [General]
I know the G14 2024 is new and I see you're integrating awesome features for it and the G16 '24...   I noticed that fan control will overshoot what I set it to when heavy load/temps rise up and then it settles back down close (but not exact) RPMs I set it to in Ghelper.   

This might be because you haven't integrated fan calibration in Ghelper on the '24 models maybe?    My '23 G14 doesn't overshoot and respects the RPMs I set it to.      Wasn't sure if I should post this as an issue or just here in General as it's not a breaking issue but mostly I can't dial in the temperature targets to RPM..  Most loads are still making my fans ramp and down for no reason which is annoying to hear..  But awesome job with the GPU TDP and OLED dimming already added!   SO MUCH THANKS.

-Div

**Comment by seerge** [comment]
@Noisrevid it's not because of that, it's cause BIOS would spin fans how it wants :) 

Only purpose of calibration is to measure max RPM (and adjust Y-axis on a chart accordingly), under the hood fan curve is set in % (same as in AC), and fans are always controlled by BIOS.

**Comment by Noisrevid** [comment]
@seerge I was afraid you'd say that ..    Ahh well.. i'll have to keep messing around with the curves, it's much harder to dial this '24 model in than my '22/'23 ones were.    Thank you!

**Reply by seerge** [reply]
I also have G14 2024 now, and by my observation even if I set fans to be off till 60C bios sometimes decides to spin them up at 57C for example.

I assume this is something asus engineers did to prevent overheating in a new slim case ...

2022 model behaves much better in this case

**Reply by Noisrevid** [reply]
I gotcha - very awesome that you have the '24 G14 too!  One other thing i'm noticing is Ghelper won't respect any changes in my CPU settings unless I set them first, then close Ghelper.. then reopen it.   Then i'll get the settings I want..  Then again, so far it seems to be only affecting undervolt and CPU temperature settings.. It seems to respect my Power Limit settings on the main CPU page..  just anything under advanced isn't working until I restart Ghelper.

**Reply by seerge** [reply]
@Noisrevid do you use latest https://github.com/seerge/g-helper/releases/tag/v0.156 ? I have noticed that in both AC and G-Helper current BIOS sometimes refuses to reset custom power limits when you go back to default modes. And added a ... "fix" :)

**Comment by Noisrevid** [comment]
Yes this is on the latest 0.156..    sometimes when I hit "Apply" on undervolt Ghelper auto restarts..  but it doesn't always do this.   And it never changes the CPU temp limit unless I restart Ghelper.    I would be fine if you even added an option under settings to "Restart Ghelper on undervolting/powerlimits/temps" :)    Seems to fix it every time for me as long as I do that.   Then again, most times I set things and leave it alone so when I dial everything in, I guess it will be a nonissue.

**Reply by seerge** [reply]
@Noisrevid undervolting needs admin rights. If app is not launched as admin - it would restart itself as such when you click Apply on undervolting.

But as soon as it's launched as admin it would also set itself to autorstart with windows with admin permissions, so you won't need to do that manually.

Why do you relaunch app so often ? Are you starting it in some unorthodox way ?

**Reply by Noisrevid** [reply]
I've always just started it like normal, double click it :)    ..   But right now i'm re-launching it often to tune my undervolt since i'm not stable -20 so far.   Otherwise I normally don't ever need to relaunch Ghelper.    Once I figure out my stable undervolt I should be good to go..  And then i'll figure out a good fan curve setup.  Now that you've added GPU temp control, that was the icing on the cake for me since I will no longer need Afterburner!    You've created quite the perfect app for Asus owners!

**Reply by seerge** [reply]
If frequently relaunch app (and also use undervolting all the time) you can just create a shortcut to exe, and set in shortcut settings to run as admin. Then it will ask for permissions on start.

I intentionally don't do it by default, cause most users don't undervolt, and admin permissions scare people.

**Reply by Noisrevid** [reply]
Good to know, I didn't realize it needed admin permissions, explains why it wasn't working properly until I did that - thank you.   Also, speaking of undervolting, I was able to get a 18,700 score in CB23!   Quite a bit faster than these go in default form - much more of a performance gain than my '23 G14 got from even a -30.

**Reply by seerge** [reply]
You have better unit than me, I'm getting ~ 18400 or so https://www.reddit.com/r/ZephyrusG14/comments/1b799mh/g14_2024_4070_some_initial_benchmarks/

**Reply by Noisrevid** [reply]
Well only when I max the fans out!    I'll never run it with max fans, far too loud for me haha.. but otherwise i'm very satisfied with the '24 G14..  such a sleek powerful unit, my favorite Asus ROG unit by far.

**Reply by Noisrevid** [reply]
Also from your Reddit post, you say you only hit 90c and still get a 18,400..  you have a better paste job than I do..  Mine pegs 95c about halfway through the CB23 run with max fans.

**Comment by Noisrevid** [comment]
Also, i'm not using AC at all.. only Ghelper.
